### PR TITLE
github-actions: support label after being merged

### PR DESCRIPTION
**Pull request by v1v**
## What is the problem this PR solves?

Fixes a corner case when the PRs have been labeled after being merged with the supported list of active branches.

## How to test this PR locally

In the CI

## Design Checklist

- [ ] I have ensured my design is stateless and will work when multiple fleet-server instances are behind a load balancer.
- [ ] I have or intend to scale test my changes, ensuring it will work reliably with 100K+ agents connected.
- [ ] I have included fail safe mechanisms to limit the load on fleet-server: rate limiting, circuit breakers, caching, load shedding, etc.

## Checklist

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `./changelog/fragments` using the [changelog tool](https://github.com/elastic/fleet-server#changelog)

### CODE REVIEW

**Comment by prodsecmachine** [comment]
### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/beats/pr-checks/4f1a18fa-cbe5-42be-9b71-b7b03c829651?refs=6ca82fafeff0960c878cf49439fa22818054b473&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/beats/pr-checks/4f1a18fa-cbe5-42be-9b71-b7b03c829651/license?refs=6ca82fafeff0960c878cf49439fa22818054b473&source=prChecksComment)

File: .github/workflows/backport-active.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     # Only run if the PR was merged (not just closed) and has one of the backport labels
     if: |
       github.event.pull_request.merged == true && 
-      contains(toJSON(github.event.pull_request.labels.*.name), 'backport-active-')
+      ( contains(toJSON(github.event.pull_request.labels.*.name), 'backport-active-') || startsWith(github.event.label.name, 'backport-active-') )
```
Comment (v1v) [inline .github/workflows/backport-active.yml]:
```suggestion
      ( 
        (github.event.action == 'closed' && contains(toJSON(github.event.pull_request.labels.*.name), 'backport-active-')) || 
        (github.event.action == 'labeled' && startsWith(github.event.label.name, 'backport-active-'))
      )
```

File: .github/workflows/backport-active.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
     # Only run if the PR was merged (not just closed) and has one of the backport labels
```
Comment (v1v) [inline .github/workflows/backport-active.yml:15]:
```suggestion
    # Only run if the PR was merged (not just closed) and has one of the backport labels
    # or has been added afterwards.
```

**Review by oakrizan — APPROVED** [review]

**Review by fr4nc1sc0-r4m0n — APPROVED** [review]
LGTM
